.github/workflows/sycl-containers.yaml

```diff
@@ -68,3 +68,29 @@ jobs:
             ghcr.io/${{ github.repository }}/${{ matrix.file }}:${{ matrix.tag }}
           build-args: ${{ matrix.build_args }}
 
+  build_all_deps_image:
+    if: github.repository == 'intel/llvm'
+    name: Build and Push Docker Image - Build With Drivers
+    runs-on: ubuntu-22.04
+    permissions:
+      packages: write
+    needs: build_and_push_images
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - name: Build and Push Container
+        uses: ./devops/actions/build_container
+        with:
+          push: ${{ github.event_name != 'pull_request' }}
+          file: ubuntu2204_intel_drivers
```

Comment:
sorry how is this different from the `_drivers` container we already generate?

Reply:
No worries! The difference is that this container uses the build image as its base image.

Reply:
Also, another note is this container is built outside of the matrix that builds the rest of the containers because it is dependent on the build image being built (It's added as a prereq for this new job). In general I was unsure what the best way to create this container would be so I was looking for feedback/ideas.

Reply:
@sarnex Actually I was wondering, the existing containers seem to have similar dependencies on each other (intel drivers container requires base container). Is it ever an issue that the intel drivers container would usually be built off of the previous base container, rather than the one being built in the current workflow? 

I ask because that's the reason I extracted this new container into a separate job.

Reply:
I'm not a total expert on Github CI but it looks like the build and drivers are both part of the matrix, so there is no dependency between them, so any issue we could describe with the new container you're adding here could in theory happen between existing images, so i would say we can probably add it to the matrix for now

Reply:
Ok, changed it to be built inside of the matrix like the rest of the containers